.azure-pipelines/scripts/RemoveSignatureScript.ps1

```diff
@@ -0,0 +1,68 @@
+function Remove-ThirdPartySignatures() {
+	<#
+  		.SYNOPSIS
+    	  The script is used to perform signature removal of third party assemblies
+
+      .PARAMETER SigntoolPath
+        Path to signtool.exe
+  		.PARAMETER LayoutRoot
+    	  Parameter that contains path to the _layout directory for current agent build
+	#>
+  [CmdletBinding()]
+  param(
+      [Parameter(Mandatory = $true)]
+      [string]$SigntoolPath,
+      [Parameter(Mandatory = $true)]
+      [string]$LayoutRoot)
+
+  $failedToUnsign = New-Object Collections.Generic.List[String]
+  $succesfullyUnsigned = New-Object Collections.Generic.List[String]
+  $filesWithoutSignatures = New-Object Collections.Generic.List[String]
+	$filesCounter = 0
```

Comment:
Nit: could you please align rows for this cmdlet?

Reply:
sure, done